addon/-legacy-private/system/store.js

```diff
@@ -833,6 +872,24 @@ Store = Service.extend({
       options,
     };
 
+    if (DEBUG) {
+      if (this._generateStackTracesForTrackedRequests === true) {
+        let trace;
+
+        try {
+          throw new Error(`Trace Origin for scheduled fetch for ${modelName}:${id}.`);
+        } catch (e) {
+          trace = e;
+        }
+
+        // enable folks to discover the origin of this findRecord call when
+        // debugging. Ideally we would have a tracked queue for requests with
+        // labels or local IDs that could be used to merge this trace with
+        // the trace made available when we detect an async leak
+        pendingFetchItem.trace = trace;
```

Comment:
eventually we should thread this through, and it super supersede the `_trackAsyncRequestStart` trace